src/main/java/bubbles/TaskList.java

```
@@ -0,0 +1,38 @@
+package bubbles;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    public TaskList(){
+        this.tasks = new ArrayList<>();
+    }
+    public TaskList(ArrayList<Task> tasks){
+        this.tasks = tasks;
+    }
+
+    public void add(Task task){
+        tasks.add(task);
+    }
+
+    public void remove(int index){
+        tasks.remove(index);
+    }
```

Nice simple remove method here!
Should we consider adding bounds checking before calling tasks.remove(index)?

Without validation, passing an invalid index would throw an unchecked IndexOutOfBoundsException.
Maybe we could throw a custom exception or return a boolean to indicate success/failure, which might give better control over error handling.